src/rdkafka.c

```diff
@@ -270,7 +271,28 @@ void rd_kafka_log0 (const rd_kafka_conf_t *conf,
         rd_kafka_log_buf(conf, rk, level, fac, buf);
 }
 
+rd_kafka_resp_err_t rd_kafka_oauthbearer_set_token(rd_kafka_t *rk,
+                const char *token_value, int64_t md_lifetime_ms,
+                const char *md_principal_name,
+                const char **extensions, size_t extension_size,
+                char *errstr, size_t errstr_size) {
+#if WITH_SASL_OAUTHBEARER
+        return oauthbearer_set_token(rk, token_value, md_lifetime_ms,
+                md_principal_name, extensions, extension_size,
+                errstr, errstr_size);
+#else
+        return RD_KAFKA_RESP_ERR__NOT_IMPLEMENTED;
+#endif
+}
 
+rd_kafka_resp_err_t rd_kafka_oauthbearer_set_token_failure(rd_kafka_t *rk,
+                const char *errstr) {
+#if WITH_SASL_OAUTHBEARER
+        return oauthbearer_set_token_failure(rk, errstr);
+#else
+        return RD_KAFKA_RESP_ERR__NOT_IMPLEMENTED;
```

Comment:
Can't since there is not return buffer into which the error message can go (`errstr` is an input rather than an output here).  Maybe `errstr` should be renamed to something else to avoid this input vs. output confusion?  Will leave alone in the absence of further feedback.

Reply:
That's my bad. Leave as is.